### PR TITLE
Safia

### PR DESCRIPTION
**Pull request by New-Arday**

### CODE REVIEW

File: src/server/controllers/movie.js [hunk 1]
```diff
@@ -1,31 +1,42 @@
...
-    } catch (e) {
-        return res.status(401).json({ error: 'Invalid token provided.' })
-    }
-
-    const createdMovie = null;
-
-    res.json({ data: createdMovie });
+  console.log("request: ", req.body);
+  try {
+    const token = req.headers.authorization.substring(7);
```
Comment (elvisonob) [inline src/server/controllers/movie.js:16]: Hi, what does this 'req.headers.authorization.substring' do?
I am not sure i used it in mine.

Comment (BrandonV21) [inline src/server/controllers/movie.js:16]: The method substring, what does that do?

File: src/server/controllers/user.js [hunk 2]
```diff
@@ -1,39 +1,49 @@
...
+  const { username, password } = req.body;
+  const hashedPassword = await bcrypt.hash(password, 10);
+  const createdUser = await prisma.user.create({
+    data: {
+      username: username,
+      password: hashedPassword,
+    },
+  });
+
+  res.json({ data: createdUser.username });
```
Comment (BrandonV21) [inline src/server/controllers/user.js:18]: Well structured, great job! I can easily see what the code is doing.

File: src/client/App.jsx [hunk 3]
```diff
@@ -1,30 +1,87 @@
...
+            console.log("created movie: ", json);
+            const updateMovieList = movies.map(function (movie) {
+              if (movie === data.json) {
+                return setMovies(updateMovieList);
+              }
+            });
+          } else {
+            console.log("Invalid response code: ", res.status);
+            console.log("Invalid response data: ", json);
+          }
```
Comment (Khateeb1111) [inline src/client/App.jsx:78]: Great use of if and else statements

**Pull request closed without merging**